features/custom-fonts.js

```diff
@@ -2,7 +2,7 @@ chrome.storage.sync.get("font", async function (obj) {
     var name = obj['font']
 var style = document.createElement('style')
 style.innerHTML = `
-@import url('https://fonts.googleapis.com/css2?family=${name.replaceAll(' ', '+')}:wght@200;300;400;500;600;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=${name.replace(/[^a-zA-Z]+/g, '').replaceAll(' ', '+')}:wght@200;300;400;500;600;700&display=swap');
 
 * {
     font-family: '${name}', sans-serif;
```

Comment:
...wouldn't this part still be subject to xss
just use innerText, it makes your life so much simpler

Reply:
> ...wouldn't this part still be subject to xss
> just use innerText, it makes your life so much simpler

It’s CSS, not JavaScript.

Reply:
`name` could be set to `</style><script>alert("xss")</script><style>`, right?

Reply:
> `name` could be set to `</style><script>alert("xss")</script><style>`, right?

That’s not the issue- what if I set the font to something like `');body{display:none;}`?

Reply:
they would both be problematic
my point is, the bug is still here

Reply:
Oh I understand, thanks